sample-compose-fragments/src/main/res/navigation/nav_graph.xml

```diff
@@ -34,7 +34,7 @@
 
     <fragment
         android:id="@+id/atomsFragment"
-        android:name="uk.gov.hmrc.sample_compose_fragments.presentation.fragment.AtomsFragment"
+        android:name="uk.gov.hmrc.sample_compose_fragments.presentation.fragment.atoms.AtomsFragment"
         android:label="@string/title_atoms"
         tools:layout="@layout/fragment_atoms"/>
```

Comment:
Please can we split this out into a `nav_graph_atoms` then include it here instead, so we have a similar structure to the main app? Then the `TextFragment` declaration can live inside the atoms graph.